tests/src/test/java/org/jboss/windup/tests/application/WindupArchitectureMediumBinaryModeProcyonTest.java

```diff
@@ -1,26 +0,0 @@
-package org.jboss.windup.tests.application;
```

Comment:
Are we removing the test because Procyon doesn't work anymore?

Reply:
Sometime ago we had a conversation about this particular test and we decided to exclude it from execution. Since the test was already being excluded I decided to remove it. Not sure if procyon doesn't work anymore, although probably we can use Fernflower exclusively.